test_data/dataset_defs/ndvi_flipped_y_axis.json

```diff
@@ -0,0 +1,329 @@
+          "218": [35, 61, 5, 255],
+          "219": [34, 61, 4, 255],
+          "220": [33, 60, 3, 255],
+          "221": [32, 59, 2, 255],
+          "222": [31, 59, 1, 255],
+          "223": [30, 58, 0, 255],
+          "224": [29, 57, 0, 255],
+          "225": [29, 57, 0, 255],
+          "226": [28, 56, 0, 255],
+          "227": [28, 55, 0, 255],
+          "228": [27, 54, 0, 255],
+          "229": [26, 54, 1, 255],
+          "230": [26, 53, 1, 255],
+          "231": [25, 52, 1, 255],
+          "232": [24, 52, 1, 255],
+          "233": [24, 51, 1, 255],
+          "234": [23, 50, 1, 255],
+          "235": [23, 49, 1, 255],
+          "236": [22, 49, 1, 255],
+          "237": [21, 48, 1, 255],
+          "238": [21, 47, 1, 255],
+          "239": [20, 47, 2, 255],
+          "240": [19, 46, 2, 255],
+          "241": [19, 45, 2, 255],
+          "242": [18, 44, 2, 255],
+          "243": [18, 44, 2, 255],
+          "244": [17, 43, 2, 255],
+          "245": [16, 42, 2, 255],
+          "246": [16, 41, 2, 255],
+          "247": [15, 41, 2, 255],
+          "248": [14, 40, 2, 255],
+          "249": [14, 39, 2, 255],
+          "250": [13, 39, 3, 255],
+          "251": [13, 38, 3, 255],
+          "252": [12, 37, 3, 255],
+          "253": [11, 36, 3, 255],
+          "254": [11, 36, 3, 255],
+          "255": [0, 0, 0, 255]
+        },
+        "noDataColor": [0, 0, 0, 0],
+        "defaultColor": [0, 0, 0, 0]
+      },
+      "opacity": 1.0
+    },
+    "provenance": {
+      "citation": "Nasa Earth Observations, MODIS Vegetation Index Products",
+      "license": "https://earthdata.nasa.gov/collaborate/open-data-services-and-software/data-information-policy",
+      "uri": "https://modis.gsfc.nasa.gov/data/dataprod/mod13.php"
+    }
+  },
+  "metaData": {
+    "type": "GdalMetaDataRegular",
+    "resultDescriptor": {
+      "dataType": "U8",
+      "spatialReference": "EPSG:4326",
+      "measurement": {
+        "type": "continuous",
+        "measurement": "vegetation"
+      },
+      "time": {
+        "start": "2014-01-01T00:00:00.000Z",
+        "end": "2014-07-01T00:00:00.000Z"
+      },
+      "bbox": {
+        "upperLeftCoordinate": [-180.0, 90.0],
+        "lowerRightCoordinate": [180.0, -90.0]
+      }
+    },
+    "params": {
+      "filePath": "test_data/raster/modis_ndvi/flipped_axis_y/MOD13A2_M_NDVI_%_START_TIME_%_flipped_y.tiff",
+      "rasterbandChannel": 1,
+      "geoTransform": {
+        "originCoordinate": {
+          "x": -180.0,
+          "y": -90.0
+        },
+        "xPixelSize": 0.1,
+        "yPixelSize": 0.1
+      },
+      "width": 3600,
+      "height": 1800,
+      "fileNotFoundHandling": "NoData",
+      "noDataValue": 0.0
+    },
+    "timePlaceholders": {
+      "%_START_TIME_%": {
+        "format": "%Y-%m-%d",
+        "reference": "start"
+      }
+    },
+    "dataTime": {
+      "start": "2014-01-01T00:00:00.000Z",
+      "end": "2014-07-01T00:00:00.000Z"
```

Comment:
should be 2014-02-01

Reply:
[682bfce](https://github.com/geo-engine/geoengine/pull/602/commits/682bfce1f0a6c100a4f8e7e152aa43e377dc62e2)